vmm_core/virt_whp/src/device.rs

```diff
@@ -181,7 +181,7 @@ fn parse_probed_bars(probed_bars: [u32; 6]) -> [u32; 6] {
     let mut i = 0;
     while i < probed_bars.len() {
         bar_flags[i] = probed_bars[i] & 0xf;
-        if probed_bars[i] & cfg_space::BarEncodingBits::TYPE_64_BIT.bits() != 0 {
+        if probed_bars[i] & cfg_space::BarEncodingBits::TYPE_64_BIT.into_bits() != 0 {
```

Comment:
Can we instead do `BarEncodingBits::from(probed_bars[i]).type_64_bit()`

Reply:
And so on elsewhere. It seems weird to try to preserve the old-style code for a few bits.

Reply:
I guess I'm saying, please just finish the refactor so we're not in this weird half state.